config/config.go

```diff
@@ -431,6 +431,38 @@ type HealthConfig struct {
 	Rate []Rate `yaml:"rate,omitempty" json:"rate,omitempty"`
 }
 
+// Versions version ranges
+type Versions struct {
+	MeshName     string `yaml:"meshName"`
+	VersionRange []struct {
+		MeshVersion string `yaml:"meshVersion"`
+		KialiLow    string `yaml:"kialiLow"`
+		KialiHigh   string `yaml:"kialiHigh"`
```

Comment:
I would prefer this more descriptive. "KialiLow" and "KialiHigh" doesn't really tell you what these are. These should be something like "KialiMinimumVersion" and "KialiMaximumVersion":
```suggestion
		KialiMinimumVersion    string `yaml:"kialiMinimumVersion"`
		KialiMaximumVersion    string `yaml:"kialiMaximumVersion"`
```

Reply:
I will update these names, but I will update related codes for it is using somewhere else.

Reply:
Done